### PR TITLE
feat(issues): add async headers for issues

### LINKED ISSUE #348: Async header for issue pages
Add two async headers for issue's pages, i.e. first header shows a issue name and second header show a issue's name, buttons to send and schedule issue and a dropdown with action for delete issue.

image

### PR DESCRIPTION
**Pull request by zouxuoz**
#375, #393, #392, #391 should be merge first

resolve #348

### CODE REVIEW

File: src/routes/campaigns/routes/issues/modules/campaigns/__tests__/creators.test.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  FETCH_CAMPAIGN_REQUEST,
+  FETCH_CAMPAIGN_SUCCESS,
+  FETCH_CAMPAIGN_FAILURE,
+  DESTROY_CAMPAIGN_REQUEST,
+  DESTROY_CAMPAIGN_SUCCESS,
+  DESTROY_CAMPAIGN_FAILURE,
+  SEND_CAMPAIGN_REQUEST,
+  SEND_CAMPAIGN_SUCCESS,
+  SEND_CAMPAIGN_FAILURE,
+  SEND_AT_CAMPAIGN_REQUEST,
```
Comment (rosendi) [inline src/routes/campaigns/routes/issues/modules/campaigns/__tests__/creators.test.js]: SEND_CAMPAIGN_AT_REQUEST, etc... :-)

File: src/routes/campaigns/routes/issues/routes/new/index.js [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+import React from 'react';
+import { injectIntl, intlShape } from 'react-intl';
+import NewCampaignHeader from './components/NewCampaignHeader';
+import NewCampaignContentContainer from './containers/NewCampaignContentContainer';
 
-export default backboneContainer(Page);
+const NewCampaign = ({ intl, ...props }) => (
+  <div>
+    <NewCampaignHeader intl={intl} />
+    <NewCampaignContentContainer {...props} />
```
Comment (rosendi) [inline src/routes/campaigns/routes/issues/routes/new/index.js]: NewCampaignContainer

File: src/routes/campaigns/routes/issues/routes/show/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+const TransformCampaign = asyncContainer(cb => require.ensure([], (require) => {
+  cb(null, require('./routes/transform').default);
+}));
+
+const CampaignRoutes = ({ pattern, params: { id } }) => (
+  <CampaignContainer campaignId={id}>
+    <Match pattern={`${pattern}/new`} component={NewCampaign} />
+    <Match pattern={`${pattern}/edit`} component={EditCampaign} />
+    <Match pattern={`${pattern}/transform`} component={TransformCampaign} />
```
Comment (rosendi) [inline src/routes/campaigns/routes/issues/routes/show/index.js]: A new line in the studio.

**Comment by radiansz** [comment]
lgtm :dagger:

**Comment by rosendi** [comment]
lgtm

**Pull request merged**